### PR TITLE
feat: add recommend article boot flag criteria

### PR DESCRIPTION
**Pull request by rebelchris**
This feature will get the flag from flagsmith.
If it's set to false we evaluate if the users profile matches the set amount of reputation.

### CODE REVIEW

**Review by idoshamun — CHANGES_REQUESTED** [review]
I suggest changing the name of this flag to `submit_article` or even `submit_community_links`.
Another important note, you set a new environment variable `RECOMMEND_ARTICLE_TRESHOLD`. But don't set in production as well. You should use Pulumi for that. 
We might want to generalize the review process that if an engineer changes the environment variables it should be reflected in production settings.
I love the fact that you decided to override the flags property!

**Comment by rebelchris** [comment]
@idoshamun Refactored to submitArticle, also ran the pulumi config. 🙌

File: infra/Pulumi.prod.yaml [hunk 1]
```diff
@@ -4,6 +4,8 @@ config:
   gateway:debeziumDbUser:
     secure: AAABAD3UsX0N4DL09nVRVJZ2uM/6FaENmEOzbwj8XFNfLZxVb9KGuQ==
   gateway:env:
+    SubmitArticleTreshold:
```
Comment (idoshamun) [inline infra/Pulumi.prod.yaml:7]: This should be camelCase. It is safe to apply this change

```suggestion
    submitArticleTreshold:
```

**Review by idoshamun — CHANGES_REQUESTED** [review]
Oh! I'm missing also some tests

**Review by idoshamun — APPROVED** [review]

**Pull request merged**